docs/user/editions/budgie/configuration.md

```diff
@@ -241,50 +241,48 @@ Click one of the images displayed on the right to choose a pre-installed backgro
 
 ## Desktop Icons
 
-With Budgie, start by opening Raven, either by clicking on its icon in the far right of the system tray or using the keyboard shortcut `Super + A`. Next, click the cog icon for Budgie Settings.
+Desktop Icons are enabled by default when Budgie is installed for the first time. To check or toggle this setting, open the Budgie Desktop Settings app by selecting "Budgie Desktop Settings" from the Budgie Menu, or from the pop-up menu when right-clicking on a blank spot on the desktop.
```

Comment:
"..enabled by default when Budgie is installed for the first time." is redundant. I would suggest something like:
Desktop Icons are enabled by default. To disable them, open...